src/styles/index.scss

```diff
@@ -1,3 +1,155 @@
+/* #region generic */
+html {
+  font-family: "Roboto", sans-serif;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+/* #endregion*/
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid black;
```

Comment:
Consider using a neutral color for the border

Reply:
Removed border, it was a property for local testing, to check whether I got the size right